src/screens/Exchange/Exchange/Exchange.js

```diff
@@ -245,6 +245,8 @@ const TouchableSwapIcon = styled.TouchableOpacity`
   align-self: center;
 `;
 
+const swapIconHitSlop = { top: 20, bottom: 20, left: 50, right: 50 };
```

Comment:
Could you check if we have something already for this? Have a look in `common.js` for `hitSlop20`